IronFrame.Shared/Utilities/JobObject.cs

```diff
@@ -291,9 +314,12 @@ private void SetJobLimits(NativeMethods.JobObjectExtendedLimitInformation extend
 
                 Marshal.StructureToPtr(extendedLimit, extendedInfoPtr, false);
 
-                if (!NativeMethods.SetInformationJobObject(handle, NativeMethods.JobObjectInfoClass.ExtendedLimitInformation, extendedInfoPtr, length))
+                if (
```

Comment:
Yeah, we are using Resharper and yeah we were sloppy in committing those formatting changes. I'll slim this down tomorrow.